api/doc/openapi.yaml

```diff
@@ -7,53 +7,96 @@ info:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   version: 0.1.0
+  contact:
+    name: The Diem Core Contributors
+    url: https://github.com/diem/diem
 tags:
-  - name: info
-    description: API for getting the latest ledger information.
-  - name: accounts
-    description: API for getting account resources, modules / events.
+  - name: general
+    description: General information
   - name: transactions
-    description: API for getting, creating, and submitting transactions.
+    description: Access to transactions
+  - name: accounts
+    description: Access to account resources and modules
   - name: events
-    description: API for getting events.
+    description: Access to events
+  - name: transaction_model
+    x-displayName: Transaction
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/Transaction"/>
+  - name: account_resource_model
+    x-displayName: AccountResource
+    description: |
+      <SchemaDefinition schemaRef="#/components/schemas/MoveResource"/>
```

Comment:
nit: I don't have as much context as I would like, but why the significant different between name, x-displayName, and the schemaRef URL. This continues below on other entries

1. account_resource_model vs AccountResource vs MoveResource
2. account_module_model vs AccountModule vs MoveModuleBytecode

Reply:
MoveResource is a shared data structure, it can be used as Account resource, or other place.
Here I used AccountResource and AccountModule is mapping to the API response.
Similar reason for MoveModuleBytecode and AccountModule.

You can see I only declared couple models here, they will be rendered as a sidebar menu item in the redocly HTML UI.
I think for people looking up document, they probably more care or start look from the root data structures that API used at root of the response object.
Thus I think the name should match the API names.